branch Component* was accepted but did not progress the lexer

If I just leave the autocomplete fields as they are, giving an invalid cron, I get a panic:

```
panic: /Users/gak/src/ftlchickenegg/chicken/chicken.go:1:9: branch Component* was accepted but did not progress the lexer at /Users/gak/src/ftlchickenegg/chicken/chicken.go:1:9 (" ")
```

```go
package chicken

import (
	"context"
)

//ftl:cron Minutes Hours DayOfMonth Month DayOfWeek
func Name(ctx context.Context) error {
	// TODO: Implement
	return nil
}
```

It's also in the wrong position.

- [ ] Fix the panic
- [ ] Report the correct line:col

--- a/backend/schema/metadatacronjob.go
+++ b/backend/schema/metadatacronjob.go
@@ -11,7 +11,7 @@ import (
 type MetadataCronJob struct {
 	Pos Position `parser:"" protobuf:"1,optional"`
 
-	Cron string `parser:"'+' 'cron' Whitespace @(' ' | ~EOL)*" protobuf:"2"`
+	Cron string `parser:"'+' 'cron' Whitespace @(' ' | ~EOL)+" protobuf:"2"`
 }
 
 var _ Metadata = (*MetadataCronJob)(nil)

--- a/internal/cron/pattern.go
+++ b/internal/cron/pattern.go
@@ -40,7 +40,7 @@ var (
 type Pattern struct {
 	Duration   *string     `parser:"@(Number ('s' | 'm' | 'h'))"`
 	DayOfWeek  *DayOfWeek  `parser:"| @('Mon' | 'Tue' | 'Wed' | 'Thu' | 'Fri' | 'Sat' | 'Sun')"`
-	Components []Component `parser:"| @@*"`
+	Components []Component `parser:"| @@+"`
 }
 
 func (p Pattern) String() string {

